lib/model/list_model.dart

```diff
@@ -14,13 +15,16 @@ class ListModel {
 
   final items = IsarLinks<Item>();
 
+  ItemType getDefaultItemType() => items.lastOrNull?.itemType ?? ItemType.text;
```

Comment:
```suggestion
  ItemType get lastItemType => items.lastOrNull?.itemType ?? ItemType.text;
```
This is where it would be cleaner to use a [getter](https://dart.dev/language/methods#getters-and-setters). We can also make the name more accurate.